Note about us english (#1578)
spryker/spryker-docs: docs/scos/user/intro-to-spryker/contribute-to-the-documentation/edit-documentation-via-pull-requests.md

For information about the structure of the Spryker documentation, as well as style and formatting rules, see [Style, syntax, formatting, and general rules](/docs/scos/user/intro-to-spryker/contribute-to-the-documentation/style-formatting-general-rules.html).

> Note that contributions to Spryker documentation can be accepted only in US English.

Author: ```suggestion
Note that Spryker documentation is written in US English.
```
